tests/test_other.py

```diff
@@ -3523,7 +3523,31 @@ def test_warn_dylibs(self):
       self.assertContainedIf(warning, err, suffix in shared_suffixes)
 
   def test_symbol_map(self):
-    UNMINIFIED_HEAP8 = 'var HEAP8 = new '
+    def get_symbols_lines(symbols_file):
+      self.assertTrue(os.path.isfile(symbols_file), "Symbols file %s isn't created" % symbols_file)
+      # check that the map is correct
+      with open(symbols_file) as f:
+        symbols = f.read()
+      lines = [line.split(':') for line in symbols.strip().split('\n')]
+      return lines
+
+    def get_minified_middle(symbols_file):
+      minified_middle = None
+      for minified, full in get_symbols_lines(symbols_file):
+        # handle both fastcomp and wasm backend notation
+        if full in ['_middle', 'middle']:
```

Comment:
Now that fastcomp is gone we can simplify this:
```suggestion
        if full == 'middle':
```